app/apollo-provider.tsx

```diff
@@ -30,7 +30,8 @@ function makeClient() {
             }),
             httpLink
           ])
-        : httpLink
+        : httpLink,
+    connectToDevTools: true
```

Comment:
I think `connectionToDevTools` may be better to enable that setting only in the development.

Reply:
I thought same at first,
but apparently [official](https://github.com/apollographql/apollo-client-devtools) says it's also ok to use this in prod dev ??? Is there acceptable condition ?
Anyway just added info